ApiTransform.ps1

```diff
@@ -483,16 +507,37 @@ Function Update-SwaggerObject
                     # Map operationIds
                     If ($AttributePath -like '*.operationId')
                     {
-                        If (($global:OperationIdMapping).Contains($ThisObject.operationId))
+                        If ($inputobject.'x-internal')

```

Comment:
If the input object's x-internal flag is set to true, remove the object from swagger

Reply:
I dont think this is necessary any more.